Frontend/MainWindow.cs

```diff
@@ -256,5 +262,23 @@ private void OpenDataDirectoryToolStripMenuItem_Click(object sender, EventArgs e
         {
             Process.Start(".");
         }
+
+        private async Task CheckForUpdates()
+        {
+            var updateLocation = ConfigurationManager.AppSettings["UpdateLocation"];
+
+            if(string.IsNullOrEmpty(updateLocation))
+            {
+                return;
+            }
+
+            // ToDo: tidy this up, allowing for using local folder and GitHub, can be done better, perhaps with DI and named instances
+            // pattern match should also be regex to allow for different url like http and www.
+            // NOTE: GitHub is untested at this point
```

Comment:
Is there any reason not to use github initially?

Reply:
I changed the configuration to use github release and changed the version to 0.4.1 so we can create a new release, this will need to happen before we use the installer.